test/hooks/useActions.spec.js

```diff
@@ -1,6 +1,8 @@
+/*eslint-disable react/display-name*/
```

Comment:
Since we have this in each file, can we just add this to a .eslintrc file instead?

Reply:
You could, but then you won't get the warning (which is legitimate) for the production code, which is probably not ideal.

Reply:
I mean a separate file in the test or test/hooks path. That scopes it to just these files.

Reply:
TIL eslint supports nested configs...

My latest commit removes the per-file disable comment and [adds a `.eslintrc` file](https://github.com/reduxjs/react-redux/pull/1259/files#diff-c5976c613641831504f0ebe88740bbce) to the `test/hooks` directory.